changelog/unreleased/pull-5170

```diff
@@ -1,6 +1,22 @@
-BugFix: Prevent Windows VSS event log 8194
+Bugfix: Prevent Windows VSS event log 8194 from being created with `restic backup --use-fs-snapshot`
```

Comment:
The title must be shorter than 80 characters.

Something like `Bugfix: Prevent Windows VSS event log 8194 warnings for backup with VSS snapshot` would work.